backoffice/app/category/category.controllers.js

```diff
@@ -604,14 +633,6 @@ CategoryControllers.controller("CategoryListCtrl", [
         }
         getMenus();
 
-        $scope.getImage = function (category) {
-            if(category && category.img) {
-                const nameImg = category.img.replace('medias/category/', '');
-                return window.location.origin + "/images/category/max-80/" + category._id + "/" + nameImg;
-            }
-            return ;
-        }
-
         function getMenus()
         {
             CategoryV2.list({PostBody: {filter: {['ancestors.0']: {$exists: false}}, populate: ["children"], sort: {displayOrder: 1}, structure: '*', limit: 99}}, function (response)
```

Comment:
Maybe use this instead of '*' for the structure, the size of each request update is really insane
check if nothing breaks
```js
{
    PostBody: {
        filter: {
            ['ancestors.0']: {$exists: false}
        },
        populate: ["children"],
        sort: {displayOrder: 1},
        structure: {
            _id: 1,
            children: 1,
            ancestor: 1,
            active: 1,
            code: 1,
            nodes: 1,
            translation: 1
        },
        limit: 99
    }
}
```

Reply:
Done !